index.js

```diff
@@ -74,22 +71,26 @@
 
     // ----- Helpers ---------------------------------------------------------------------------------------------------------------
     /**
-     * Construct any additional resources.
-     * Currently this occurs post rendering, and is used to initialize javascript widgets that affect the display.
-     * @private
-     * @method _plug
+     * Construct any additional resources when the view is attached to the DOM.
+     * Currently this is used to initialize javascript widgets that affect the display.
+     * @method attach
+     * @override
      */
-    _plug: function() {
-      this._brecTableInit();
-      this._brecWidgetsInit();
+    attach: function(element) {
```

Comment:
you can probably use the _attached() and _detached() instead of overriding the attach and detach methods